ibis/backends/base/sql/compiler/translator.py

```diff
@@ -345,7 +345,11 @@ def _category_label(expr):
     if op.nulls is not None:
         stmt = stmt.else_(op.nulls)
 
-    return stmt.end().name(expr._name)
+    result = stmt.end()
+    if expr.has_name():
+        result = result.name(expr.get_name())
```

Comment:
I guess we don't have a test that hits this branch. Probably fine to get rid of the conditional.

Reply:
Breaks tests if I remove the has_name() check.

Reply:
Does the postgres build report to codecov?